jenkins-integration/beaker/install/shared/install_deps.rb

```diff
@@ -0,0 +1,9 @@
+step "Configure epel" do
+  tmp_module_dir = master.tmpdir('configure_epel')
+  on(master, puppet('module', 'install', 'stahnma-epel', '--codedir', tmp_module_dir))
+  on(master, puppet('apply', '-e', "'include epel'"))
```

Comment:
Nuking in the case of success makes sense, and yeah, I think it's probably reasonable to try to keep this from interfering with normal production code.